Aml 223 create new organisation

robinoconnor opened the pull request.

dashton82 commented:
Ive done a commit removing the Interface on the orchestrator. Also look at implementing QueryBaseTest for GetEmployerAgreementTemplateTests

robinoconnor commented:
I've added the validator and used the QueryBaseTest as the based class for the test. I've also re-merge master into the latest version of the branch and fixed and issues with the new code on master
